From 948bc69839a1dc546847ecf90af7c3c97296b654 Mon Sep 17 00:00:00 2001
From: "Arun Kumar K.R" <akumarkr@codeaurora.org>
Date: Wed, 19 Dec 2012 18:23:45 -0800
Subject: [PATCH] hwcomposer: Add HWC_BLIT compositiontype value

- HWC_BLIT this layer will be handled in the HWC, using a blit engine
- setting this will inform Surfaceflinger to flip FB_TARGET
Conflicts:
	include/hardware/hwcomposer_defs.h

Change-Id: I25d19ec41b57863d451c3e390109c912faf1381c
Conflicts:
include/hardware/hwcomposer_defs.h
---
 include/hardware/hwcomposer_defs.h | 7 +++++--
 1 file changed, 5 insertions(+), 2 deletions(-)

diff --git a/include/hardware/hwcomposer_defs.h b/include/hardware/hwcomposer_defs.h
index 9a52436..3728ce6 100644
--- a/include/hardware/hwcomposer_defs.h
+++ b/include/hardware/hwcomposer_defs.h
@@ -115,8 +115,11 @@ enum {
        cursor overlay hardware. hwcomposer will also all async position updates
        of this layer outside of the normal prepare()/set() loop. Added in
        HWC_DEVICE_API_VERSION_1_4. */
-    HWC_CURSOR_OVERLAY =  5
- };
+    HWC_CURSOR_OVERLAY =  5,
+
+    /* this layer will be handled in the HWC, using a blit engine */
+    HWC_BLIT = 6,
+};
 /*
  * hwc_layer_t::blending values
  */

